set-web/src/main/java/eu/europeana/set/web/service/impl/UserSetServiceImpl.java

```diff
@@ -581,7 +589,7 @@ public CollectionPage buildCollectionPage(UserSet userSet, LdProfiles profile, i
 		CommonLdConstants.COLLECTION);
 
 	int startIndex = pageNr * pageSize;
-	CollectionPage page = new CollectionPage(userSet, partOf, startIndex);
+	CollectionPage page = new CollectionPage(partOf, startIndex);
```

Comment:
it is likely that we need to take in account the page here ... or in better in an apply profile method ...

Reply:
so shall i add the Userset in the constructor ? as in applyprofile we pass only userset and profile as a paramater